mantidproject/mantid
# Export algorithm dialogs to python

Used sip to export algorithm GenericDialog and InterfaceManager to python.

**To test:**

Code review. Open dialogs via exported InterfaceManager. Start AlgorithmSelector with `run_test_app.py` and execute algorithms. Check logs to see that it works.

Fixes #21281

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
